package.json

```diff
@@ -21,6 +21,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/lodash.shuffle": "^4.2.6",
```

Comment:
we can just import the entire lodash library, it'll be help for other operations as well